app.js

```diff
@@ -8,7 +8,6 @@ const environment = process.env.NODE_ENV || 'development';
 const configuration = require('./knexfile')[environment];
 
 var indexRouter = require('./routes/index');
-var papersRouter = require('./routes/api/v1/papers');
```

Comment:
Thank you for deleting this and the test. This should improve our test coverage.